src/shared/platform/interfaces.ts

```diff
@@ -0,0 +1,51 @@
+export type IRawPlatformFile = {
+  LaunchBox: IRawPlatform
+}
+
+export type IRawPlatform = {
+  Game?: IRawGameInfo | IRawGameInfo[],
+  AdditionalApplication?: IRawAdditionalApplicationInfo | IRawAdditionalApplicationInfo[]
+}
+
+/**
+ * This interface represents a Game as stored in Platform XMLs
+ * Comments represent their XML data types and IPureGameInfo counterparts
+ * (NOTE: Best not to mess with the casing of the fields)
+ */
+export type IRawGameInfo = {
+  ID: string;                   // (String)
+  Title?: string;               // (String)
+  Series?: string;              // (String)
+  Developer?: string;           // (String)
+  Publisher?: string;           // (String)
+  DateAdded?: string;           // (DateTime)
+  Platform?: string;            // (String)
+  Broken?: boolean;             // (Boolean)
+  Hide?: boolean;               // (Boolean) - Extreme
+  PlayMode?: string;            // (String)
+  Status?: string;              // (String)
+  Notes?: string;               // (String)
+  Genre?: string;               // (String) - Tags / Genres
+  Source?: string;              // (String)
+  ApplicationPath?: string;     // (String)
+  CommandLine?: string;         // (String) - Launch Command
+  ReleaseDate?: string;         // (Nullable<DateTime>)
+  Version?: string;             // (String)
+  OriginalDescription?: string; // (String)
+  Language?: string;            // (String)
+}
+
+/**
+ * This interface represents an Additional Application as stored in Platform XMLs
+ * Comments represent their XML data types and IAdditionalApplicationInfo counterparts
+ * (NOTE: Best not to mess with the casing of the fields)
```

Comment:
I think the links to the Launchbox docs should be kept, at least until we stop using their XML format